protocol/SignalingProtocol.proto

```diff
@@ -248,7 +248,11 @@ message SdkMetric {
         VIDEO_DISCARDED_PPS = 47;
         VIDEO_PLIS_SENT = 48;
         VIDEO_RECEIVED_JITTER_MS = 49;
+        VIDEO_INPUT_HEIGHT = 60;
```

Comment:
@XHatan Do we want this to send back to tincan?

Reply:
optional.

Reply:
Confirming with Tianyu, actually he need the video frame height and width related metrics for another project, so it would be great if we send them back to tincan.